internal/operators/ocs/validations.go

```diff
@@ -12,13 +12,13 @@ import (
 
 type Config struct {
 	OCSMinimumCPUCount                int64             `envconfig:"OCS_MINIMUM_CPU_COUNT" default:"18"`
-	OCSMinimumRAMGB                   int64             `envconfig:"OCS_MINIMUM_RAM_COUNT" default:"57"`
+	OCSMinimumRAMGiB                  int64             `envconfig:"OCS_MINIMUM_RAM_GiB" default:"57"`
```

Comment:
Will changing the names in the envconfig affect ant tests, for eg, ci?

Reply:
Yes, but they aren't used in CI test

Reply:
@rewantsoni Do we need to update QE about it? 
@lalon4 FYI

Reply:
Yes @gamli75, QE can use these envconfig

Reply:
it means that QE will need to update their tests after this change will be merged to master. Please **don't** push it before 05/05 - the next deployment to production.
@lalon4 are you using that envconfig in your testing?

Reply:
we use default envconfig to deploy PSI / minikube - so we are good